tensorboard/backend/application.py

```diff
@@ -277,7 +277,10 @@ def _serve_plugins_listing(self, request):
     response = {}
     for plugin in self._plugins:
       start = time.time()
-      response[plugin.plugin_name] = plugin.is_active()
+      response[plugin.plugin_name] = {
+          'enabled': plugin.is_active(),
+          'es_module_path': plugin.es_module_path() or '',
```

Comment:
Hmm, identifying the empty string with “no module” seems confusing: I’d
expect the empty string to mean that the ES module is hosted at simply
`/data/plugins/PLUGINS/`. Can we just drop the `or ''` and let the field
be `null`?